HUD/SetMouseCursorStyle.md

```diff
@@ -0,0 +1,33 @@
+---
+ns: HUD
+aliases: ["0x8DB8CFFD58B62552","_SET_CURSOR_SPRITE", "_SET_MOUSE_CURSOR_SPRITE"]
+---
+## SET_MOUSE_CURSOR_STYLE
+
+```c
+// 0x8DB8CFFD58B62552
+void SET_MOUSE_CURSOR_STYLE(int style);
+```
+
+Changes the mouse style.
+
+Available styles are:
+| ID  | Description         |
+| :-: | :------------------:|
```

Comment:
Not sure if this will parse correctly. Pretty sure tables in MD need a space on `---:|` -> `---: |`

Reply:
Done!